packages/one-app-bundler/package.json

```diff
@@ -38,7 +38,7 @@
   ],
   "license": "Apache-2.0",
   "peerDependencies": {
-    "babel-eslint": "7.x || 8.x"
+    "babel-eslint": "7.x || 8.x || 10.x"
```

Comment:
This is to match the version used by one-app, as mismatched peerDeps cause install failures in node 16